code/game/mecha/combat/durand.dm

```diff
@@ -42,3 +51,12 @@
 	infra_luminosity = 8
 	force = 40
 	wreckage = /obj/structure/mecha_wreckage/durand/old
+
+/obj/mecha/combat/durand/old/examine_more(mob/user)
+	..()
+	. = list()
+	. += "<i>A relic of a mech, once produced by Defiance Arms nearly one hundred years ago. \
```

Comment:
Time advances. This description will become wrong with time. Use absolute time, not relative time.

Reply:
Resolved this in my most recent commit